Products/ZenWidgets/browser/quickstart/templates/quickstart_macros.pt

```diff
@@ -344,7 +344,8 @@ Ext.onReady(function() {
                     labelStyle: 'padding-left: 20px',
                     fieldLabel: 'Community Strings',
                     name: 'snmpcommunities',
-                    value: DEFAULT_SNMPCOMMUNITIES
+                    value: DEFAULT_SNMPCOMMUNITIES,
+                    allowBlank: false
```

Comment:
We don't use the quickstart macros since the wizard was rewritten. See Products/ZenUI3/browser/resources/js/zenoss/quickstart directory. I am not sure if this problem is still applicable.